source/sessions/driver-sessions.rst

```diff
@@ -1111,8 +1110,17 @@ ensure that they close any explicit client sessions and any unexhausted cursors.
 
 13. To confirm that implicit sessions only allocate their server session after a successful connection checkout
 
-    * Create a MongoClient with maxPoolSize=1, and attach a command succeeded listener
-    * Initiate 3 concurrent operations, e.g. collection.insertOne(doc)
+    * Create a MongoClient with the following options: ``maxPoolSize=1``, ``retryableReads=true``, and ``retryableWrites=true``
```

Comment:
Read retryability is not relevant since that feature doesn't make use of sessions

Nit: the option is called `retryWrites`.  It's also enabled by default so not sure we need to even mention it.  But your call.

Reply:
Doh! on the options name, I figure it's safer to call out what should be enabled despite it being the expected default we want to make sure that the retryable path is tested. fixed the retryRead mention